src/main/java/ch/njol/skript/expressions/ExprNodeValue.java

```diff
@@ -0,0 +1,149 @@
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.classes.Parser;
+import ch.njol.skript.config.EntryNode;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.ParseContext;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.Feature;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Value")
+@Description({
+	"Returns the value of a node in a loaded config.",
+})
+@Examples({})
+@Since("INSERT VERSION")
+public class ExprNodeValue extends SimplePropertyExpression<Node, Object> {
+
+	static {
+		Skript.registerExpression(ExprNodeValue.class, Object.class, ExpressionType.PROPERTY,
+				"[the] %*classinfo% value [at] %string% (from|in) %node%",
+				"[the] %*classinfo% value of %node%",
+				"[the] %*classinfo% values of %nodes%",
+				"%node%'s %*classinfo% value",
+				"%nodes%'[s] %*classinfo% values"
+		);
+	}
+
+	private boolean isSingle;
+	private ClassInfo<?> classInfo;
+	private Parser<?> parser;
+	private @Nullable Expression<String> pathExpression;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] expressions, int pattern, Kleenean isDelayed, ParseResult parseResult) {
+		if (!this.getParser().hasExperiment(Feature.SCRIPT_REFLECTION))
+			return false;
+		@NotNull Literal<ClassInfo<?>> format;
+		switch (pattern) {
+			case 0:
+				this.isSingle = true;
+				format = (Literal<ClassInfo<?>>) expressions[0];
+				this.pathExpression = (Expression<String>) expressions[1];
+				this.setExpr((Expression<? extends Node>) expressions[2]);
+			break;
+			case 1:
+				this.isSingle = true;
+			case 2:
+				format = (Literal<ClassInfo<?>>) expressions[0];
+				this.setExpr((Expression<? extends Node>) expressions[1]);
+				break;
+			case 3:
+				this.isSingle = true;
+			default:
+				format = (Literal<ClassInfo<?>>) expressions[1];
+				this.setExpr((Expression<? extends Node>) expressions[0]);
+		}
+		this.classInfo = format.getSingle();
+		if (classInfo.getC() == String.class) // don't bother with parser
+			return true;
+		if (classInfo.getParser() == null) {
+			Skript.error("The type '" + classInfo.getName() + "' has no parser.");
+			return false;
+		}
+		if (!classInfo.getParser().canParse(ParseContext.CONFIG)) {
+			Skript.error("The type '" + classInfo.getName() + "' cannot be used to parse config values.");
+			return false;
+		}
+		this.parser = classInfo.getParser();
```

Comment:
```suggestion
		this.parser = classInfo.getParser();
		if (this.parser == null || !this.parser.canParse(ParseContext.CONFIG)) {
			Skript.error("The type '" + classInfo.getName() + "' cannot be used to parse config values.");
			return false;
		}
```
I think using the same error message is fine.